### ISSUE TITLE
Error message that doesn't happen in Vim

### ISSUE THREAD

**Issue by chdiza** [macvim-dev/macvim#1069]
In line 7 of syntax/cfg.vim, there is a space between the function name and the left parenthesis.

In MacVim 8.2.1377, when I open a file of type `cfg` (even with `mvim --clean`), I receive the following error message:

```
Error detected while processing modelines[50]..FileType Autocommands for "*"..Sy
ntax Autocommands for "*"..function <SNR>4_SynSet[25]..script /Users/chdiza/Applica
tions/MacVim.app/Contents/Resources/vim/runtime/syntax/cfg.vim:
line    7:
E121: Undefined variable: exists
```

I also get it if I run MacVim's embedded Vim inside the console.

But importantly I do not get it if I run regular console vim from Bram's repo.

**Comment by ichizok** [comment]
I confirmed this happened in also Vim (not MacVim) 8.2.1377 and fixed in 8.2.1378.

**Comment by chdiza** [comment]
It's fixed now after the latest merge from upstream.
